### PR TITLE
Import from team

### PR DESCRIPTION
**Pull request by amberwebb**

### CODE REVIEW

**Comment by nathanielvelez** [comment]
:+1:

File: src/collections/members.coffee [hunk 1]
```diff
@@ -145,3 +145,27 @@ exports.canEditItem = (member, team, item) ->
     item.href is member.href
   else
     item.links.member.href is member.href
+
+exports.importMembersFromTeam = (memberIds, teamId, callback) ->
```
Comment (matthopson) [inline src/collections/members.coffee]: Curious, what does this method return when successful?

File: docs/collections/members.md [hunk 2]
```diff
@@ -292,3 +294,40 @@ if(teamsnap.canEditItem(member, team, availability)){
...
+---
+
+
+<a id="loadImportableMembers"></a>
+## `loadImportableMembers(userId, includeArchivedTeams)`
+Creates a list of importable members by current member's userId.
+
+## Params
+* `userId`: [id] - `userId` of current member.
+* `includeArchivedTeams`: [bool] - include or exclude members from archived teams.
```
Comment (matthopson) [inline docs/collections/members.md]: optional?

File: src/collections/members.coffee (hunk 1 above)
Comment (amberwebb) [inline src/collections/members.coffee]: This imports the members into the current team.

Comment (matthopson) [inline src/collections/members.coffee]: Right, but what I'm wondering is what's in the response body... is it a list of member that are bing imported? Or the team? Or what?

File: src/collections/members.coffee [hunk 3]
```diff
@@ -145,3 +145,27 @@ exports.canEditItem = (member, team, item) ->
...
+    `memberIds`'
+  unless @isItem teamId
+    throw new TSArgsError 'teamsnap.importMembersFromTeam', "must provide
+     a teamId"
+  if @isItem teamId
+    teamId = teamId.id
+  params = sourceMemberIds: memberIds, destinationTeamId: teamId
+
+  @collections.members.exec('importFromTeam', params)
+    .pop().callback callback
```
Comment (matthopson) [inline src/collections/members.coffee]: So if we're returning an array of members that were added, do we want to pop() that list? Or return the whole array of members? I feel like we want to return all the members.

Comment (amberwebb) [inline src/collections/members.coffee]: Why do you think so?

Comment (matthopson) [inline src/collections/members.coffee]: Doesn't it make sense to get the full return vs a piece of it? We may not do this in our app, but if someone working with v3 was expecting to see the entire list of imported members in the response, but didn't with the SDK, we'd be tricking them up. You may, for instance, want to take the response and display a confirmation: "The following members were imported..." or something.

**Comment by amberwebb** [comment]
@matthopson Please review again.

**Comment by matthopson** [comment]
Looks good :+1:

**Pull request merged**